docs/openapi.yml

```diff
@@ -278,6 +278,53 @@ paths:
               schema:
                 $ref: '#/components/schemas/datasets'
 
+  /datasources:
+    get:
+      summary: List all datasources
+      description: Returns a list of datasources.
+      tags:
+        - Datasources
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/datasources'
+    post:
+      summary: Create a new datasource
+      description: Creates a new datasource and returns the created datasource
```

Comment:
Minor: Let's be consistent with how we describe endpoints. I recommend looking over the descriptions for endpoints that have been already defined.

Reply:
I updated the create description to be much more informative, similar to what is done for namespace creation and job creation. Also updated for punctuational consistency. Is there anything else I should be looking at to make it consistent with the rest of the file?